rust/hyperlane-base/src/contract_sync/cursor.rs

```diff
@@ -225,11 +225,11 @@ impl ForwardMessageSyncCursor {
                 .retrieve_dispatched_block_number(self.cursor.sync_state.next_sequence)
                 .await
             {
-                debug!(next_block = block_number, "Fast forwarding next block");
+                info!(next_block = block_number, "Fast forwarding next block");
```

Comment:
wondering if this is intentional here and down below

Reply:
was meant only for debugging, thanks for pointing out